pkg/adapter/mtping/runner_test.go

```diff
@@ -31,9 +31,12 @@ import (
 	"knative.dev/eventing/pkg/logging"
 )
 
+const THREE_SECONDS_TILL_NEXT_MIN_CRON_JOB = 60 - 3
```

Comment:
One very last comment from my side: constants do not follow the all-uppercase naming pattern from C/C++ due to the uppercase==public behavior of Go.

It's a test so not very important, but I thought it would be worth mentioning.

Reply:
I can't help but continue to leak my Java background